pytest_fixtures/component/satellite_auth.py

```diff
@@ -343,8 +343,8 @@ def module_enroll_idm_and_configure_external_auth(module_target_sat):
     ipa_host.disenroll_idm()
 
 
-@pytest.mark.external_auth
-@pytest.fixture
+@pytest.mark.external_auth()
```

Comment:
correct, each mark is a decorator. adding the parenthesis for them is more for forcing consistency. though we could likely force consistency without the parenthesis. this is up for debate in this PR.
https://docs.astral.sh/ruff/rules/pytest-incorrect-mark-parentheses-style/

Reply:
I'm definitely in favor of the parens for consistency. The only downside is it looks a little more cluttered visually, but it feels substantially more consistent to be specifically declaring that these are all functions, even if there aren't any arguments

Reply:
Yeah I mean if its really accepting params thats fine but we can keep it optional if it isnt taking params, looks bit weird !

Reply:
In the general world(Learning material etc), pytest marks without params are represented without parenthesis so good to follow that so that any new person contributes in robottelo should not have confusion !

I dont want to press to much on it though!